src/panel/error_panel.js

```diff
@@ -3,9 +3,6 @@ import Panel from '../libs/panel'
 
 function Error() {
   this.message = ''
-  listen('error_h', (message = '') => {
-    this.showMessage(message)
-  })
```

Comment:
I think this error panel is not used anymore.  
Could we remove it from the main app panel ? Or delete the file completely ?

Reply:
yep ill clear it